### DISCUSSION TITLE
Fan mode missing from automation actions

### DISCUSSION THREAD

**Discussion by nmkloster** [veista/nilan#114] [Q&A] [answered]
When setting up an automation the only option for Nilan HVAC is hvac mode and not fan mode
![image](https://github.com/veista/nilan/assets/20794464/71856800-4802-46e3-a1a3-d87c2ca632b5)
![image](https://github.com/veista/nilan/assets/20794464/b1ab0611-aabb-40d6-89e5-9b90775263d8)

**Comment by veista** [comment]
Hi, Nilan devices do not have a fan mode.

**Comment by nmkloster** [comment]
![image](https://github.com/veista/nilan/assets/20794464/34da9e5a-8028-40c3-a350-2a3a9c84c7bf)
I can change fan mode on the hvac card. Or fan speed.

**Comment by veista** [comment, marked as answer]
https://github.com/veista/nilan/discussions/28#discussioncomment-7060643

**Reply by nmkloster** [reply]
Thank you. I think I'm too green to fully get it. I can't get the fan control service but I'm on modbus 21. I can control power levels instead to attain the same effect.
